style.css

```diff
@@ -1 +1,366 @@
-/*  Add your styling here */
+body {
+    display: grid;
+    margin: 0 auto;
+    grid-template-columns: 1rem auto 0.5rem 0.5rem auto 1rem;
+    grid-template-rows: 130px 1rem 190px 1rem 200px 1rem 160px 1rem 160px 1rem 150px;
+    grid-template-areas: "header  header header header header header" ". . . . . ." ". welcome welcome welcome welcome ." ". . . . . ." ". hero  hero  hero  hero ." ". . . . . ." ". birthday . . cupcakes ." ". . . . . ." ". wedding  . . gluten ." ". . . . . ." "footer footer footer footer footer footer";
+    margin: 0;
+}
+
+.header {
+    background-color: #f1d6d4;
+    /* For browsers that do not support gradients */
+    background-image: linear-gradient(to bottom right, #ecddf3, #ede7d2, #ffd5de, #e6dbeb, #ede7d2, #f9ddf6);
```

Comment:
I love this! Very clever and the effect is so pretty.
Can you think how you can use CSS variables to make a reusable colour palette for your CSS?